pycontw/pycontw-frontend
# feat: history page with imgs

## Types of changes
* **New feature**

## Description
blocked by https://github.com/pycontw/pycontw-frontend/pull/240

## Steps to Test This Pull Request
1. open /2022/about/history
2. check history's text: [zh-hant, tw]
3. check images' by adjusting the screen width 

## Expected behavior

## Related Issue
#232
